packaging/update-docs.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+case "$1" in
+	bleed)
+		exit
+		;;
+	next | playtest-*)
+		TAG="playtest"
+		;;
+	master | release-*)
+		TAG="release"
+		;;
+	*)
+		echo "Unknown branch: $1"
+		exit
+		;;
+esac
+
+echo "Updating docs.openra.net"
+
+SSH_KEY="$HOME"/.ssh/id_rsa
+
+mkdir -p "$(dirname "$SSH_KEY")"
+openssl aes-256-cbc -k "$KEY" -in ssh.enc -d -out "$SSH_KEY"
+chmod 0600 "$SSH_KEY"
+
+rm -rf "$HOME/openra-docs"
+git clone git@github.com:OpenRA/docs.git "$HOME/openra-docs"
+
+mono --debug ../OpenRA.Utility.exe all --docs "$1" > "${HOME}/openra-docs/api/${TAG}/traits.md"
```

Comment:
Have you tested this script?

I would expect this line to fail because we haven't yet made the `${HOME}/openra-docs/api/${TAG}/` directory.

Reply:
https://github.com/OpenRA/docs/tree/master/api/playtest already exists. The latest update of the repository was done using this script as a test case if I remember correctly. It has been 2 months now.